tests/abi/testabi_rel.c

```diff
@@ -181,6 +186,13 @@ static void test_below (void)
 	keySetName (k2, "system:/a/a/a/a/a/a");
 	succeed_if (keyIsBelow (k1, k2) == 1, "should be below");
 
+	keySetName (k1, "/");
+	keySetName (k2, "user:/something");
+	succeed_if (keyIsBelow (k1, k2) == 1, "should be below");
+
+	keySetName (k1, "user:");
```

Comment:
this test should succeed according to @markus2330 suggestion

Reply:
Do you mean that `keySetName (k1, "user:")` succeeds? (I am okay with requiring `keySetName (k1, "user:/")`). And better to always check return values!

```suggestion
	succeed_if(keySetName (k1, "user:/") == 6, "could not set name");
```